sharding.md

```diff
@@ -350,7 +350,7 @@ Here is an empirical measurement of AllGather bandwidth on a TPU v5e 8x16 slice.
 
 {% include figure.liquid path="assets/img/all-gather-bandwidth.png" class="img-small" caption="<b>Figure:</b> empirical bandwidth and estimated link bandwidth for TPU v5e during an AllGather. BW in orange is the actual bytes per second AllGathered, while the blue curve shows the empirical unidirectional link bandwidth calculated according to the known cost of the collective." %}
 
-Note both that we achieve only about 95% of the peak claimed bandwidth (`4.5e10`) and that we achieve this peak at about 10MB, which when 16-way sharded gives us about 500kB per device (*aside: this is much better than GPUs).
+Note both that we not only achieve about 95% of the peak claimed bandwidth (`4.5e10`) but we also achieve this peak at about 10MB, which when 16-way sharded gives us about 500kB per device (*aside: this is much better than GPUs).
```

Comment:
Could you fix the italics on the aside? Also maybe say "Note that we not only achieve...but also that we achieve this"